server.go

```diff
@@ -640,6 +641,85 @@ func (sp *serverPeer) OnGetBlocks(_ *peer.Peer, msg *wire.MsgGetBlocks) {
 	}
 }
 
+// locateBlocks returns the hashes of the blocks after the first known block in
+// locators, until hashStop is reached, or up to a max of
+// wire.MaxBlockHeadersPerMsg block hashes.  This implements the search
+// algorithm used by getheaders.
+//
+// NOTE: This is a btcsuite extension ported from
+// github.com/decred/dcrd.
```

Comment:
i'd remove this part of the comment. this isn't a new feature, just some code reorganization that dcrd did to make this feature available to call by the rpc server.